### PR TITLE
KU-1062 ROCK for cluster-autoscaler

### PR DESCRIPTION
**Pull request by maci3jka**
code moved from #1

### CODE REVIEW

File: rockcraft.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+parts:
+  cluster-autoscaler:
+    plugin: nil
+    source: https://github.com/kubernetes/autoscaler.git
+    source-type: git
+    source-tag: cluster-autoscaler-1.24.0
+    source-depth: 1
+#    build-packages:
+#      - build-essential
```
Comment (bschimke95) [inline rockcraft.yaml]: nit: obsolete comment?

**Review by bschimke95 — COMMENTED** [review]
LGTM, leaving it to our Rocks experts @berkayoz or @addyess to give the final approval.

File: rockcraft.yaml (hunk 1 above)
Comment (maci3jka) [reply rockcraft.yaml]: yep forgot abut those

Comment (maci3jka) [inline rockcraft.yaml]:
```suggestion
```

**Review by berkayoz — APPROVED** [review]
We will need to adjust the tests to be like what we've done in the `cert-manager-rock` but for now LGTM.

**Pull request merged**